### ISSUE TITLE
I cannot build the dll

### ISSUE THREAD

**Issue by pedermark** [ChoZenTime/aristois-pasted-by-choZen#11]
I get a error no such file or directory found"d3xd9" i get the same error like 23 times. What can i do?

**Comment by KernCS** [comment]
https://lmgtfy.com/?q=C1083%3A+Cannot+open+include+file%3A+%27d3dx9.h%27%3A+No+such+file+or+directory

**Comment by NickelWW** [comment]
> https://en.lmgtfy.com/?q=C1083%3A+Cannot+open+include+file%3A+%27d3dx9.h%27%3A+No+such+file+or+directory

that doesn't goddam help everything on google doesn't do anything

**Comment by KernCS** [comment]
> > https://en.lmgtfy.com/?q=C1083%3A+Cannot+open+include+file%3A+%27d3dx9.h%27%3A+No+such+file+or+directory
> 
> that doesn't goddam help everything on google doesn't do anything

Legit the second [link](https://www.unknowncheats.me/forum/programming-for-beginners/311095-include-file-d3dx9-file-directory.html) on google thats popping up

**Comment by NickelWW** [comment]
yeah it didn't help I found a solution after like an hour tho (It was similar)

**Comment by xsoma** [comment]
> yeah it didn't help I found a solution after like an hour tho (It was similar)

1, if it helps: download this: https://www.microsoft.com/en-us/download/details.aspx?id=6812

2, after download build the project. if its still not working:

3, goto "C:\Program Files (x86)\Microsoft DirectX SDK (June 2010)\Include", open your project and drag "d3dx9.h" into "Resource Files"


it works for me all the time:) hope i can help you
